app/javascript/react/components/App.js

```diff
@@ -2,6 +2,7 @@ import React from 'react'
 import ParkShowContainer from '../containers/ParkShowContainer'
 import { BrowserRouter, Route, Switch } from 'react-router-dom'
 
+
```

Comment:
In the future, please review your changes before creating a commit so that you don't commit trivial changes!